xclim/core/utils.py

```diff
@@ -71,8 +71,27 @@
     "pr": "time: sum within days",
 }
 
+T = TypeVar("T", bound=Callable)
 
-def wrapped_partial(func: Callable, suggested: dict | None = None, **fixed) -> Callable:
+
+class WrappedCallable:
+    """A wrapper around a callable that allows for attribute access."""
+
+    def __init__(self, func: T) -> None:  # noqa: D102
+        self._func = func
+
+    def __call__(self, *args, **kwargs):  # noqa: D102
+        return self._func(*args, **kwargs)
+
+    def __getattr__(self, attr):  # noqa: D105
+        return getattr(self._func, attr)
+
+    _injected: dict[str, Any]
```

Comment:
I know you told me about this, but I now understand less... What's the use of creating a full class that is never being used ? Isn't the output of `wrapped_partial` an actual callable and not a custom object ?

Reply:
This entire wrapper code has been removed, as it has not been in use for several versions (replaced by `declare_units`)